### ISSUE TITLE
Add Location

### ISSUE THREAD

**Issue by reneelynn** [Crypton/statusapp#218]
# Description

Add approximate location to your post.
# Expected result

Will post "near [city] [state] [country]" with your post.

(NOTE: see Github#176 and #178)
# Actual result

No location is being posted.
# Other information

Reported by: Renee Jackson 

Test case number: TC08 

Test run: [https://spideroak.testlodge.com/projects/14421/runs/191185?tab=2&run_section_id=165468&executed_case_id=9265228#executed_case_9265228](https://spideroak.testlodge.com/projects/14421/runs/191185?tab=2&run_section_id=165468&executed_case_id=9265228#executed_case_9265228)

**Comment by daviddahl** [comment]
Is it posting "near unknown"?

**Comment by daviddahl** [comment]
also, you might have to tell the settings to allow kloak access to location if for some reason you declined in the past

**Comment by reneelynn** [comment]
it is leaving no information at all; not even "near unknown".
This is a fresh install of Kloak using a first-time login of the staging user you set up for me earlier today. Location is enabled on my phone.

Kloak doesn't even show up on Recent Location Requests in my phone Settings.

**Comment by daviddahl** [comment]
is location for Kloak enabled?

**Comment by reneelynn** [comment]
Yes. Location is enabled for everything. If I look into Kloak's app info specifically it says the app has permission for my approximate location.

**Comment by daviddahl** [comment]
is this on Android or iOS?

**Comment by reneelynn** [comment]
Android

**Comment by reneelynn** [comment]
Just confirmed it is not working on iOS either, and iOS asked me to confirm whether I wanted to allow location services in Kloak when I selected the option. I said "yes", but my post shows only a timestamp and date.

**Comment by reneelynn** [comment]
Now iOS is finally showing locations for my posts; I would have expected it on Post A, when I selected the option, but it didn't show until Post B. Actually, it appears to only be posting with images in iOS and not with text.

**Comment by daviddahl** [comment]
WFM, so not sure what is going on here.

**Comment by reneelynn** [comment]
OK; 

Android: Can't get location working at all
iOS: Can get it working, but there's a delay before it shows up under my post-in-progress, and it appears that if I hit "post" before it shows up, then it posts without my location.

**Comment by daviddahl** [comment]
funnily enough, none of that code was touched in any way, sigh.
